docs-ref-services/powerbi-embedded.md

```diff
@@ -13,26 +13,59 @@ ms.devlang: nodejs
 ms.service: PowerBI Embedded
 ---
 
-# Azure PowerBI Embedded module for Node.js
+# Azure PowerBI Embedded package for Node.js
 
 ## Overview
 
 With the Power BI Embedded Azure service, you can integrate Power BI reports right into your node application to create or edit charts and reports.
 
 Learn more about [Power BI Embedded](https://powerbi.microsoft.com/en-us/documentation/powerbi-developer-embedding/).
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure PowerBI Embedded module for Node.js
+### Install npm module
 
-### Management
 ```bash
 npm install azure-arm-powerbiembedded
 ```
 
+### Example
 
-##Example
+Authenticate, create client and create workspace collection in an existing resource group.
 
-##Samples
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const PowerBIEmbeddedManagementClient = require('azure-arm-powerbiembedded');
 
-Explore more [sample Node.js code](https://azure.microsoft.com/resources/samples/?platform=nodejs) you can use in your apps.
\ No newline at end of file
+const creationOptions = {
+  location: 'northcentralus',
+  tags: {
+    key1: 'value1',

```

Comment:
what are the keys for? would a better variable name help clarify it? if not, maybe a small comment.